### ISSUE TITLE
Live Captions

### ISSUE THREAD

**Issue by mikewillems** [berkmancenter/frankly#330]
### Pitch

User sees captions on the screen (optionally) during 
- meeting breakouts
- during videos
- hosted event main room presentations
- live streams

### Motivation

Users should be able to get live captions for both accessibility and independence on audio hardware / quiet environment.
